lib/src/data/irma_repository.dart

```diff
@@ -505,41 +495,29 @@ class IrmaRepository {
 
   static const _iiabchannel = MethodChannel('irma.app/iiab');
 
-  Future<String> getInAppCredential() {
-    return _inAppCredentialSubject.first.then((state) => state.credentialType);
-  }
-
-  Future<void> processInactivation() async {
-    final curState = await _inAppCredentialSubject.first;
-    if (curState.pendingInactivations > 0) {
-      // If there are still inactivations to be ignored, we ignore
-      // and just decrement count
-      _inAppCredentialSubject.add(curState.copyWith(
-        pendingInactivations: curState.pendingInactivations - 1,
-      ));
-    } else {
-      // Forget about previous opening of browser
-      _inAppCredentialSubject.add(_InAppCredentialState());
-    }
-  }
-
-  // Remember that an inactivation of the app is coming due to opening the browser
-  // for issuance of the given credential type. Opening the browser inactivates the app
-  // (i.e. sets the AppLifecycleState to paused), so we must not react to that the way
-  // we normally do (e.g. go back to the previous app or show the big topleft arrow).
-  void expectInactivationForCredentialType(String type) {
-    _inAppCredentialSubject.add(_InAppCredentialState(pendingInactivations: 1, credentialType: type));
+  Future<List<String>> getCredentialsLaunchedFromStore() {
+    return _credentialStoreSubject.first.then(
+      (state) => state.launchedCredentials,
+    );
   }
 
   Future<void> openIssueURL(BuildContext context, String type) async {
     final lang = FlutterI18n.currentLocale(context)!.languageCode;
-    final translated = await _irmaConfigurationSubject.first
-        .then((irmaConfiguration) => irmaConfiguration.credentialTypes[type]?.issueUrl);
-    final url = translated?.translate(lang, fallback: '') ?? '';
+
+    final irmaConfig = await _irmaConfigurationSubject.first;
+    final cred = irmaConfig.credentialTypes[type]!;
```

Comment:
Maybe it's nice to do a null check here to prevent that the function fails unexpectedly when requesting an unknown credential ID without a clear error.